docs/4.2/features/enhanced_session_recording.md

```diff
@@ -0,0 +1,185 @@
+something like the following.
+
+```
+$ uname -a
+Linux ip-172-31-43-104.ec2.internal 4.19.72-25.58.amzn2.x86_64 x86_64 x86_64 x86_64 GNU/Linux
+```
+
+
+### Ubuntu 
+
+|       |                   | Kernel Version        |
+|-------|-------------------|-----------------------|
+| 18.10 | Cosmic Cuttlefish | 4.18 [Patch Kernel](http://www.theubuntumaniac.com/2018/11/update-install-kernel-4191-stable-on.html)  |
+| 19.04 | Disco Dingo       | 5.0 ✅                  |
+| 19.10 | Eoan Ermine       | 5.3 ✅                  |
+
+### Debian
+
+|     |                     | Kernel Version            |
+|-----|---------------------|---------------------------|
+| 9   | Debian Stretch      | 4.9.0-6 [Patch Kernel](https://wiki.debian.org/HowToUpgradeKernel) |
+| 10  | Buster              | 4.19 ✅                   |
+
+### CentOS
+|               | Kernel Version         |
+|---------------|------------------------|
+| 8.0-1905	    |            4.18.0.80 ✅  |
+
+### Red Hat 
+|                     | Kernel Version         |
+|---------------------|------------------------|
+| Enterprise Linux 8  |          4.18.0-147 ✅ |
+
+### Amazon Linux 
+We recommend using `Amazon Linux 2` to install and use Linux kernel 4.19 using
+`sudo amazon-linux-extras` install kernel-ng and rebooting your instance. 
+
+### archlinux 
+|                     | Kernel Version         |
+|---------------------|------------------------|
+| 2019.12.01          |              5.3.13 ✅ |
+
+## 2. Install BCC Tools 
+
+Run the following script to download the prerequisites to build BCC tools, building LLVM and Clang targeting BPF byte code, and then building and installing BCC tools.
+
+!!! note
+    We plan have plans to remove this step after we've moved our of our beta.
+
+```sh
+#!/bin/bash
+
+set -e
+
+if [[ $EUID -ne 0 ]]; then
+   echo "Please run this script as root or sudo." 
+   exit 1
+fi
+
+# Create a temporary to build tooling in.
+BUILD_DIR=$(mktemp -d)
+cd $BUILD_DIR
+echo "Building in $BUILD_DIR."
+
+# Install Extra Packages for Enterprise Linux (EPEL)
+yum install -y epel-release
+yum update -y
+
+# Install development tools.
+yum groupinstall -y "Development tools"
+yum install -y elfutils-libelf-devel cmake3 git bison flex ncurses-devel
+
+# Download and install LLVM and Clang. Build them with BPF target.
+curl  -LO  http://releases.llvm.org/7.0.1/llvm-7.0.1.src.tar.xz
+curl  -LO  http://releases.llvm.org/7.0.1/cfe-7.0.1.src.tar.xz
+tar -xf cfe-7.0.1.src.tar.xz
+tar -xf llvm-7.0.1.src.tar.xz
+
+mkdir clang-build
+mkdir llvm-build
+
+cd llvm-build
+cmake3 -G "Unix Makefiles" -DLLVM_TARGETS_TO_BUILD="BPF;X86" \
+  -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ../llvm-7.0.1.src
+make
+make install
+cd ..
+
+cd clang-build
+cmake3 -G "Unix Makefiles" -DLLVM_TARGETS_TO_BUILD="BPF;X86" \
+  -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ../cfe-7.0.1.src
+make
+make install
+cd ..
+
+# Install BCC.
+git clone https://github.com/iovisor/bcc.git
```

Comment:
Add the following line after `git clone https://github.com/iovisor/bcc.git`:

`(cd bcc && git checkout v0.11.0)`